### ISSUE TITLE
autochunk: option to save all simulated data to disk

### ISSUE THREAD

**Issue by caglorithm** [neurolib-dev/neurolib#139]
WIP, maybe netcdf?

**Comment by jajcayn** [comment]
yes, that would be doable.. `xarray` supports saving to netcdf with `a` option, which is append. AFAIK, each chunk would be saved with correct time dimension with `a` option, and this would result in one big nc file with stitched chunks

**Comment by caglorithm** [comment]
I'll get my hands dirty with `xarray` then and ask for your help :)
